### PR TITLE
Install rust target `wasm32-unknown-unknown` if missing

### PR DESCRIPTION
**Pull request by FirelightFlagboy**

### CODE REVIEW

**Comment by FirelightFlagboy** [comment]
https://github.com/Scille/parsec-cloud/actions/runs/5887987901

File: bindings/web/scripts/build.js [hunk 1]
```diff
@@ -105,12 +105,27 @@ function ensure_rust_target_installed() {
...
 
+function install_rust_target(target) {
+  const rustup_install_target_args = [
+    on_windows() ? "rustup.exe" : "rustup",
+    "target",
+    "install",
+    target
+  ];
+
+  console.log(`Installing rust target ${target}`);
```
Comment (mmmarcos) [inline bindings/web/scripts/build.js:120]: Maybe add some quotes as with the other messages
```suggestion
  console.log(`Installing rust target '${target}'`);
```

**Review by mmmarcos — APPROVED** [review]
LGTM

**Comment by FirelightFlagboy** [comment]
Superseded by #5026

**Pull request closed without merging**